app/src/main/java/com/tohami/newsapi/Constants.java

```diff
@@ -0,0 +1,28 @@
+package com.tohami.newsapi;
+
+public class Constants {
+    public class Prefs {
+        public static final String PREF_FILE_NAME = "news_prefs" ;
+        public static final String APP_LOCALE_KEY = "appLocale";
+        public static final String LANGUAGE_IS_SELECTED = "LANGUAGE_IS_SELECTED";
+        public static final String LANGUAGE_KEY = "language_key";
+    }
+
+    public class DateTime {
+        public static final String FORMAT_DEFAULT = "yyyy-MM-dd";
+        public static final String FORMAT_ARTICLE = "MMM dd, yyyy";
+    }
+
+    public enum Status {
+        SUCCESS , FAIL , LOADING
+    }
+
+    public class Api {
```

Comment:
Remove smiley faces this is not proper messages

Reply:
done